src/hashes/mod.rs

```diff
@@ -1,8 +1,19 @@
 // Copyright 2020 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+#[cfg(feature = "blake2b")]
+pub mod blake2b;
+
 #[cfg(feature = "curl-p")]
 pub mod curl_p;
 
 #[cfg(feature = "sha")]
 pub mod sha;
+
+#[cfg(feature = "digest")]
+#[doc(inline)]
+pub use digest::Digest;
+
+#[cfg(feature = "digest")]
+#[doc(inline)]
+pub use digest::Output;
```

Comment:
Any reason not to merge them in a single `use` ?

Reply:
nope, just overlooked